styleguide/source/_patterns/01-molecules/06-components/22-article-preview/_article-preview.scss

```diff
@@ -65,4 +65,44 @@
       padding-right: 0;
     }
   }
+}
+
+.flexbox .topic_topic-articles .topic_article-preview.topic_article-preview-small {
```

Comment:
Class names here should be updated to follow [BEM ](http://getbem.com/naming/) so that `topic_topic-articles` would be `topic__topic-articles` (double underscore) and `topic_article-preview` would be `topic__article-preview` (double underscore) and `topic_article-preview-small` should be `topic__article-preview--small` (double underscores and double hyphen).

Reply:
This seems to be happening in a number of places, in addition to this particular instance.